test/e2e/volume_create_test.go

```diff
@@ -79,6 +79,39 @@ var _ = Describe("Podman volume create", func() {
 		Expect(check.OutputToString()).To(ContainSubstring("hello"))
 	})
 
+	It("podman create and import volume", func() {
+		if podmanTest.RemoteTest {
+			Skip("Volume export check does not work with a remote client")
+		}
+
+		session := podmanTest.Podman([]string{"volume", "create", "my_vol"})
+		session.WaitWithDefaultTimeout()
+		volName := session.OutputToString()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"run", "--volume", volName + ":/data", ALPINE, "sh", "-c", "echo hello >> " + "/data/test"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"volume", "export", volName, "--output=hello.tar"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"volume", "create", "my_vol2"})
+		session.WaitWithDefaultTimeout()
+		volName = session.OutputToString()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"volume", "import", "my_vol2", "hello.tar"})
+		session.WaitWithDefaultTimeout()
+		volName = session.OutputToString()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"run", "--volume", "my_vol2:/data", ALPINE, "cat", "/data/test"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.OutputToString()).To(ContainSubstring("hello"))
+	})
+
```

Comment:
could you extend your tests to include non-zero exits as well?  for example, supplying only one arg on the command line, giving a bad input file path, output path, etc.  anything that would protect against regressions is helpful.